expo-zustand-styled-components/src/components/RepoCard/RepoCard.tsx

```diff
@@ -14,33 +14,34 @@ import { StarLogo } from '../Icons/StarLogo';
 import { colors } from '../../utils/style-variables';
 import { Text, TouchableWithoutFeedback } from 'react-native';
 
-import { Repository } from '../../interface/repositories.interface';
-
 interface RepoCardProps {
-  repo: Repository;
   star?: boolean;
+  full_name: string;
+  visibility: 'public' | 'private';
+  description: string;
+  forks_count: number;
+  stargazers_count: number;
+  language: string;
+  updated_at: Date;
 }
 
-const RepoCard = ({ repo, star }: RepoCardProps) => {
-  const {
-    id,
-    name,
-    owner,
-    description,
-    stargazers_count,
-    forks_count,
-    language,
-    updated_at,
-    visibility,
-  } = repo;
-
+const RepoCard = ({
+  full_name,
+  visibility,
+  description,
+  forks_count,
+  stargazers_count,
+  language,
+  updated_at,
+  star,
+}: RepoCardProps) => {
   return (
-    <Card key={id}>
+    <Card>
       <Content>
         <Heading>
-          <Link>Repo name here</Link>
+          <Link>{full_name}</Link>
```

Comment:
On press of this, it should take use to the repo page, currently, there is nothing addressing that.
What do you plan?

Reply:
The page doesn't exist now, I'll add it to the repos page ticket